internal/lib/sandbox/sandbox.go

```diff
@@ -186,8 +186,8 @@ func (s *Sandbox) SeccompProfilePath() string {
 }
 
 // AddIP stores the ip in the sandbox
```

Comment:
Absolutely non-blocking: I wonder why the linter doesn't complain about the missing 's'.